.changeset/thin-candles-cross.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fix overrides.extends order when including same rules
```

Comment:
[suggest] According to [our convention](https://stylelint.io/maintainer-guide/pull-requests#merging):

```suggestion
Fixed: `overrides.extends` order when including same rules
```

Reply:
@ybiquitous done.